src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,6 +56,11 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
+        try {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (DateTimeException | IndexOutOfBoundsException e) {
+            System.out.println("Illegal date or index is out of bounds!");
```

Comment:
```suggestion
            System.out.println("Something went wrong: " + e.getMessage());
```
you should be able to see what exactly went wrong

Reply:
це круто! дякую!